### PR TITLE
Update code-splitting.md

### PR DESCRIPTION
**Pull request by mitchell-frost**
This resolves the issue #3835 

- [x] Read and sign the [CLA][1]. PRs that haven't signed it won't be accepted.
- [x] Make sure your PR complies with the [writer's guide][2].
- [x] Review the diff carefully as sometimes this can reveal issues.
- [ ] Do not abandon your Pull Request: [Stale Pull Requests][3].
- __Remove these instructions from your PR as they are for your eyes only.__

[1]: https://cla.js.foundation/webpack/webpack.js.org
[2]: https://webpack.js.org/contribute/writers-guide/
[3]: https://webpack.js.org/contribute/#pull-requests

### CODE REVIEW

**Comment by mitchell-frost** [comment]
This resolves the issue #3835

File: src/content/guides/code-splitting.md [hunk 1]
```diff
@@ -141,6 +141,14 @@ The [`dependOn` option](/configuration/entry-context/#dependencies) allows to sh
   };
 ```
 
+#### `optimization.runtimeChunk`
+
+`optimization.runtimeChunk: "single"` is needed when multiple entry points are put on a single HTML page.
```
Comment (EugeneHlushko) [inline src/content/guides/code-splitting.md:146]:
```suggestion
`optimization.runtimeChunk: 'single'` is needed when multiple entry points are being used on a single HTML page.
```

File: src/content/guides/code-splitting.md [hunk 2]
```diff
@@ -141,6 +141,14 @@ The [`dependOn` option](/configuration/entry-context/#dependencies) allows to sh
...
+
+Using multiple entrypoints per page should be avoided when possible in favor of an entrypoint with multiple imports: `entry: { page: ["./analytics", "./app"] }`. This results in better optimization and consistent execution order when using async script tags.
```
Comment (EugeneHlushko) [inline src/content/guides/code-splitting.md:148]:
```suggestion
Using multiple entrypoints per page should be avoided when possible in favor of an entrypoint with multiple imports: `entry: { page: ["./analytics", "./app"] }`. This results in a better optimization and consistent execution order when using `async` script tags.
```

File: src/content/guides/code-splitting.md [hunk 3]
```diff
@@ -141,6 +141,14 @@ The [`dependOn` option](/configuration/entry-context/#dependencies) allows to sh
...
+
+Multiple entrypoints per page are still valid to use in scenarios where HTML is generated in a dynamic matter, e. g. when components on page are unknown at compile-time and HTML page is composed dynamically depending on data.
```
Comment (EugeneHlushko) [inline src/content/guides/code-splitting.md:150]:
```suggestion
T> Multiple entrypoints per page could be used in scenarios where HTML is generated in a dynamic matter, e. g. when components on page are unknown at compile-time and HTML page is composed dynamically depending on the data.
```

File: src/content/guides/code-splitting.md [hunk 4]
```diff
@@ -141,6 +141,14 @@ The [`dependOn` option](/configuration/entry-context/#dependencies) allows to sh
...
+
```
Comment (EugeneHlushko) [inline src/content/guides/code-splitting.md:151]:
```suggestion
```

**Comment by EugeneHlushko** [comment]
Thanks!

**Comment by mitchell-frost** [comment]
Thanks!

**Pull request merged**